RELEASE.adoc

```diff
@@ -93,10 +93,12 @@ cp ./result/bin/uplc ./uplc-x86_64-linux-ghc96
 It is desirable to minimize the amount of change between `rc1` and `rc2`, because it may reduce the tests and checks that need to be performed against `rc2`.
 For instance, if `plutus-ledger-api` is the only package changed, there is no need to re-run tests on `plutus-core` or `plutus-tx`.
 Another example is if a security audit is done on `rc1`, and the changes in `rc2` do not modify the audited code, then the audit does not need to be re-done.
+- If another CHaP PR gets merged before yours it will invalidate the timestamps and you won't be able to merge: see the https://github.com/IntersectMBO/cardano-haskell-packages/blob/main/README.md[CHaP README file], in particular https://github.com/IntersectMBO/cardano-haskell-packages/blob/main/README.md#monotonically-increasing-timestamps[this section].  Read https://github.com/IntersectMBO/cardano-haskell-packages/blob/main/README.md#dealing-with-timestamp-conflicts[this section] for advice on how to deal with this problem.
+
 8. Once the CHaP PR has been merged, make a PR to update the version used in `cardano-ledger`. Example: https://github.com/IntersectMBO/cardano-ledger/pull/3563.
 - Update the version bounds in cabal files.
 - Update the CHaP index state in `cabal.project`.
-- Update the CHaP flake input with `nix flake update CHaP`.
+- Update the CHaP flake input with `nix flake lock --update-input CHaP`.
```

Comment:
This command is deprecated
<img width="591" alt="image" src="https://github.com/IntersectMBO/plutus/assets/1009751/2534208f-173c-45a3-9b7a-c31515134213">

IMO we should update our docs to use a newer version before its removed.

Reply:
Aha!  2.12.0.  I kind of assumed that nix was providing nix, but that probably doesn't make sense.  I'll update it and try again.  Somehow I've managed to do this before without any problems though.

Reply:
> I checked the cardano-ledger CONTRIBUTING document (see the link in comment number 2) and it says to use `--update-input`

We need to update CONTRIBUTING instructions too.

Reply:
OK, it works with 2.21.1.  Thanks!  I've added a warning about that to the document.

Reply:
Thank you!